src/common/utils/FocusManager.ts

```diff
@@ -226,8 +233,11 @@ export abstract class FocusManager {
                     needsFocusReset = false;
                 }
 
-                if (prevFocusedComponent && !prevFocusedComponent.removed &&
-                        !prevFocusedComponent.restricted && !prevFocusedComponent.limitedCount) {
+                if (prevFocusedComponent &&
+                    !prevFocusedComponent.removed &&
+                    !prevFocusedComponent.restricted &&
+                    !prevFocusedComponent.limitedCount &&
```

Comment:
limitedCount and limitedCountAccessible aren't booleans, so the proper check is "> 0".